nidirect_common/inc/preprocess.inc

```diff
@@ -129,3 +129,24 @@ function nidirect_common_preprocess_taxonomy_term(array &$variables) {
 
   $variables['information_services_output'] = $content;
 }
+
+/**
+ * Implements hook_preprocess_webform_actions().
+ */
+function nidirect_common_preprocess_webform_actions(array &$variables) {
+  $route_match = \Drupal::routeMatch();
+
+  if ($route_match->getRouteName() == 'entity.node.canonical') {
+    $node = \Drupal::routeMatch()->getParameter('node');
```

Comment:
Switch `\Drupal::routeMatch()`here  for `$route_match` declared on line 137?

Reply:
doh ! copy and pasta from my ksm() test!